.github/workflows/update-and-prepare-release.yml

```diff
@@ -0,0 +1,52 @@
+# TODO abort if there are no changes to the Gemfile.
+
+name: Update gems and prepare to release Docker image
+
+on:
+  repository_dispatch:
+    types:
+      - gem-released-disabled
+  workflow_dispatch:
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+    - uses: ruby/setup-ruby@v1
+
+    - id: update
+      run: script/workflows/update.sh
+      env:
+        RELEASED_GEM_NAME: ${{ github.event.client_payload.name }}
+        RELEASED_GEM_VERSION: ${{ github.event.client_payload.version }}
+
+    - name: Commit and push gem file changes
+      uses: EndBug/add-and-commit@v4
+      with:
+        add: Gemfile.lock
+        author_name: Beth Skurrie via Github Action
```

Comment:
Could use the github action user here. I probably didn't know about it when I hardcoded this.

Reply:
yep makes sense to update it now, whilst we are here